lib/ES/Util.pm

```diff
@@ -794,11 +798,6 @@ http {
         add_header 'Access-Control-Allow-Headers' 'kbn-xsrf-token';
       }
     }
-    types {
-      text/html  html;
-      application/javascript  js;
-      text/css   css;
-    }
```

Comment:
These didn't do anything because nginx expects the proxied application to set the mime type.